### PR TITLE
Fix S3 service-key script

### PR DESCRIPTION
**Pull request by mogul**
## Changes proposed in this pull request:
- Fix the example script so that it actually works.
- Note that the script must be "sourced" (run with a '. ' in front of it for maximum cross-shell compatibility).

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/BRANCH_NAME)

## Security Considerations

No impact to the system. Corrects a non-functional script in documentation which customers might copy-and-paste.

### CODE REVIEW

File: _docs/services/s3.md [hunk 1]
```diff
@@ -100,18 +100,25 @@ For example, you might want to use the [AWS Command Line Interface](https://aws.
 ```sh
 #!/bin/bash
 
+Run this script with a . in order to set environment variables in your shell
```
Comment (pburkholder) [inline _docs/services/s3.md:103]: For readability, it might be better to use `source` instead of `.`, but OK to ship as far as I'm concerned.

**Review by pburkholder — APPROVED** [review]

File: _docs/services/s3.md (hunk 1 above)
Comment (mogul) [reply _docs/services/s3.md:103]: I was going to do that, but I discovered that only the "." syntax is portable across shells per POSIX spec.

Comment (mogul) [reply _docs/services/s3.md:103]: https://stackoverflow.com/a/20097303/17138235

Comment (mogul) [reply _docs/services/s3.md:103]: https://ss64.com/bash/source.html#:~:text=source%20is%20a%20synonym%20for,available%20after%20the%20script%20completes.

**Comment by mheadd** [comment]
Changes incorporated via #2062

**Pull request merged**